src/bandersnatch/verify.py

```diff
@@ -60,11 +63,13 @@ async def get_latest_json(
         if e.status == 404:
             # A 404 means that the package has been removed from PyPI.
             # Allow function to continue, and remove package files if applicable.
-            pass
+            # write a blank json file to make the deletion process go through
+            write_fake_json(json_path)
```

Comment:
This is a bit hack, I wrote a fake json in json_path to make "clean blob" process go through, please let me know if this is not a good practice

Reply:
So you need this just so other deletion code does not error? Can we just catch an exception if the json file does not exist and skip writing out this fake JSON file all together?

Reply:
Certainly, the only downside is that the nested `if` and `try` make the indent comparely long, I'll try that.